packages/RNTester/js/examples/Modal/builder.js

```diff
@@ -0,0 +1,271 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+import React from 'react';
+import {
+  Modal,
+  Switch,
+  Text,
+  TouchableHighlight,
+  View,
+  StyleSheet,
+  Platform,
+} from 'react-native';
+import {Picker} from '@react-native-community/picker';
```

Comment:
I am not sure about using `@react-native-community/picker` here. I think it was there used in the old code when `react-native` had `Picker` component in the core but maybe now it needs to be removed.

Reply:
sure, any idea how do we implement the picker now ?

Reply:
@anku255 , I say we tackle it in the next sprint. Re implementing a community module is a ticket on its own imo

Reply:
so should I mark this convo as resolved ?